src/client/pages/Dashboard/Dashboard.tsx

```diff
@@ -26,8 +26,8 @@ const MAX_TIPS_VISIBLE = 3;
 
 function sortAndFormatChapters(items: ChapterMenuItem[]) {
   const shortNameItems = items
-    .sort((a, b) => a.title.localeCompare(b.title))
-    .map((item) => item.title.substring(0, 3));
+    .map((item) => item.title.substring(0, 3))
+    .sort((a, b) => a.localeCompare(b));
```

Comment:
Sorting after the map because sorting modifies array in place which is a side-effect